mobile/lib/modules/backup/providers/backup.provider.dart

```diff
@@ -300,7 +300,18 @@ class BackupNotifier extends StateNotifier<BackUpState> {
     Set<AssetEntity> assetsFromExcludedAlbums = {};
 
     for (var album in state.selectedBackupAlbums) {
-      var assets = await album.albumEntity.getAssetListRange(
```

Comment:
Another thought on this method, why is it modified since it doesn't related to the purpose of this PR?

The reason I am asking because this method is modified in #853